torch_xla/csrc/aten_xla_bridge.cpp

```diff
@@ -64,6 +65,16 @@ XLATensor GetXlaTensor(const at::Tensor& tensor) {
   return *xtensor;
 }
 
+std::tuple<XLATensor, XLATensor> GetPromotedXlaTensorsForBinaryOp(
```

Comment:
This belongs more to `tensor_ops.*` than here.

Reply:
Hmmmm I think it fit here better since it's taking `at::Tensor` as inputs and return `XLATensor` which is mostly what the bridge does? ;)

Reply:
You are right ... tensor_ops are ops among XLA tensors.
But they do not belong here as well. The bridge should not know about binary ops and type promotion.
Given that the only caller is aten_xla_type.cpp put this in the anon namespace there.